[2024-04-28T17:40:10.837+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T17:40:10.865+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_processing_dag.process_csv_files manual__2024-04-28T17:35:03.241671+00:00 [queued]>
[2024-04-28T17:40:10.873+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_processing_dag.process_csv_files manual__2024-04-28T17:35:03.241671+00:00 [queued]>
[2024-04-28T17:40:10.874+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-04-28T17:40:10.886+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): process_csv_files> on 2024-04-28 17:35:03.241671+00:00
[2024-04-28T17:40:10.897+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=225) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-28T17:40:10.901+0000] {standard_task_runner.py:63} INFO - Started process 227 to run task
[2024-04-28T17:40:10.898+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'csv_processing_dag', 'process_csv_files', 'manual__2024-04-28T17:35:03.241671+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/csv_dags.py', '--cfg-path', '/tmp/tmp3k9dv4cs']
[2024-04-28T17:40:10.902+0000] {standard_task_runner.py:91} INFO - Job 103: Subtask process_csv_files
[2024-04-28T17:40:10.963+0000] {task_command.py:426} INFO - Running <TaskInstance: csv_processing_dag.process_csv_files manual__2024-04-28T17:35:03.241671+00:00 [running]> on host 4cdb6c1dd19f
[2024-04-28T17:40:11.073+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_processing_dag' AIRFLOW_CTX_TASK_ID='process_csv_files' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T17:35:03.241671+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-28T17:35:03.241671+00:00'
[2024-04-28T17:40:11.075+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T17:40:11.811+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-28T17:40:11.813+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T17:40:11.824+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=csv_processing_dag, task_id=process_csv_files, execution_date=20240428T173503, start_date=20240428T174010, end_date=20240428T174011
[2024-04-28T17:40:11.883+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T17:40:11.899+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-28T17:40:11.901+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
